apps/modernization-ui/src/apps/search/investigation/InvestigationSearch.tsx

```diff
@@ -37,7 +33,7 @@ const InvestigationSearch = () => {
                         />
                     )}
                     resultsAsTable={() => <div>result table</div>}
-                    onSearch={handleSubmit}
+                    onSearch={form.handleSubmit(search)}
```

Comment:
Question: This reverts the change that was merged today: https://github.com/CDCgov/NEDSS-Modernization/pull/1560/files#

Was this not verified locally?